UI-Session-1-Assignment/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Metacube Parking System</title>
+</head>
+
+<body>
+    <!-- Div for nav bar -->
+    <div id="nav_bar">
```

Comment:
This navigation bar should be moved on each section click.

Reply:
It can be done with CSS which is the next assignment.